utils.js

```diff
@@ -274,6 +274,46 @@ function toBase64(hexString) {
   return Buffer.from(hexString, "hex").toString("base64");
 }
 
+const spdxLicenseExpressionOp = [
+  " with ",
+  " and ",
+  " or ", // Apache-2.0 OR MIT
+  "-or-later", // GPL-2.0-or-later
+  "-only" // GPL-2.0-only
+];
+
+/**
+ * Method to determine if a license is a valid SPDX license expression
+ *
+ * @param {string} license License string
+ * @returns {boolean} true if the license is a valid SPDX license expression
+ * @see https://spdx.dev/learn/handling-license-info/
+ **/
+function isSpdxLicenseExpression(license) {
+  const licenseLoweCase = (license || "").toLowerCase();
+  if (!licenseLoweCase) {
+    return false;
+  }
+
+  if (
+    spdxLicenseExpressionOp.some((op) => {
```

Comment:
Can this be simplified to check for the characters space and open bracket `(`?

Reply:
Should it check for the existence of  both characters or at least one? I have see license expressions without the `(` character.

Reply:
You are correct. I meant either.